include/ScratchViews.h

```diff
@@ -412,23 +412,22 @@ void MasterElementViews<T>::fill_master_element_views_new_me(
 template<typename T>
 ScratchViews<T>::ScratchViews(const TeamHandleType& team,
              const stk::mesh::BulkData& bulkData,
-             stk::topology topo,
-             ElemDataRequests& dataNeeded)
+             int nodesPerEntity,
```

Comment:
@alanw0 I didn't understand the motivation to switch from topo to nodesPerEntity, can you explain?